### PR TITLE
chore: support font scaling on artwork rail cards

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [] 

### Description

Resolves issue reported on slack: https://artsy.slack.com/archives/C05EQL4R5N0/p1732719691127379 (Save and Follow CTAs are cut off)

| Before Artworks Rail | Before Previously Sold Rail | After Artworks Rail | After Previously Sold Rail | After Artworks Rail Android | After Previously Sold Rail Android |
| --- | --- | --- | --- | --- | --- |
| IMAGE 2024-11-28 11:24:41 | IMAGE 2024-11-28 11:24:45 | IMAGE 2024-11-28 11:22:34 | IMAGE 2024-11-28 11:22:38 | Screenshot_1732789660 | Screenshot_1732789696 |

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix Save and Follow CTAs are cut off when using font scaling

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix Save and Follow CTAs are cut off when using font scaling - dariakoko)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5ec586e841a7068532526b69298322dd99b80e62
</p>

File: src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx [hunk 1]
```diff
@@ -106,7 +107,7 @@ const RecentlySoldArtworksRail: React.FC<RecentlySoldArtworksRailProps> = ({
             onPress?.(item, index)
           }}
           // adding 20 to make sure the estimated section is not cut off when displayed in two lines
-          containerHeight={ARTWORK_RAIL_CARD_IMAGE_HEIGHT + PixelRatio.getFontScale() * 100 + 20}
+          containerHeight={ARTWORK_RAIL_CARD_IMAGE_HEIGHT + fontScale * 100 + fontScale * 20}
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx:110]: Do you know where the 100 comes from?
```suggestion
          containerHeight={ARTWORK_RAIL_CARD_IMAGE_HEIGHT + fontScale * 120}
```

File: src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx [hunk 2]
```diff
@@ -155,14 +156,16 @@ const RecentlySoldCardSection: React.FC<RecentlySoldCardSectionProps> = ({
   highEstimateDisplay,
   performanceDisplay,
 }) => {
+  const priceFontSize = fontScale > 1 ? "sm" : "md"
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx:159]: I like this solution 👍

**Review by olerichter00 — DISMISSED** [review]
Amazing, thank you very much for adjusting the design for font scaling! 🌟

File: src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx (hunk 1 above)
Comment (MrSltun) [inline src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx:110]: Question: Was the change from `+ 20` to `* 20` intentional?

**Review by MrSltun — DISMISSED** [review]
Looks good! Thank you Daria!! ✨ 
~~I just added one question~~

File: src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx (hunk 1 above)
Comment (MrSltun) [reply src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx:110]: oops, nvm I missed the `fontScale * 20`

Comment (dariakoko) [reply src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx:110]: 100 is the height of the text component of 92px and 10px vertical margin

Actually, i need to change 100 to 102

File: src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx [hunk 3]
```diff
@@ -123,22 +125,25 @@ const RecentlySoldArtworksRail: React.FC<RecentlySoldArtworksRailProps> = ({
...
-      ListHeaderComponent={ListHeaderComponent}
-      listRef={listRef}
-      onEndReached={onEndReached}
-      onEndReachedThreshold={onEndReachedThreshold}
-      prefetchUrlExtractor={(item) => item?.artwork?.href || undefined}
-      renderItem={renderItem}
-      showsHorizontalScrollIndicator={false}
-    />
+    <>
+      <Text>{fontScale}</Text>
```
Comment (brainbicycle) [inline src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx:129]: 👀

**Pull request merged**